### ISSUE TITLE
Drawer a11y attribute typo

### ISSUE THREAD

**Issue by mbechev** [telerik/kendo-angular#4647]
### Description

The `.k-drawer ul` element currently has orientation="vertical". But according to the [a11y spec](https://www.telerik.com/kendo-angular-ui/components/layout/drawer/accessibility/aria-support_drawer#wai-aria) should be aria-orientation="vertical"

![Image](https://github.com/user-attachments/assets/87a859a0-3dc2-4cc3-9650-f6ceee9447ae)

### Steps To Reproduce

Inspect the demo:
https://www.telerik.com/kendo-angular-ui/components/layout/drawer#kendo-ui-for-angular-drawer-overview

### Actual Behavior

Wrong attribute name.

### Expected Behavior

The element to have `aria-orientation="vertical"`. Here is the KendoReact Drawer for reference.
https://www.telerik.com/kendo-react-ui/components/layout/drawer#kendoreact-drawer-overview

### Browser

Chrome

### Browser version

latest

### OS type

MacOS
